pkg/indicator/v2/bollinger_band.go

```diff
@@ -24,16 +23,15 @@ type BOLLStream struct {
 //
 //	-> calculate SMA
 //	-> calculate stdDev -> calculate bandWidth -> get latest SMA -> upBand, downBand
-func BOLL(source types.Float64Source, window int, k float64) *BOLLStream {
+func BollingerBand(source types.Float64Source, window int, k float64) *BollingerStream {
```

Comment:
is there a reason of renaming BOLL to BollingerBand?

we use BOLL because binance uses BOLL, just like EMA, we don't call it ExponentiallyMovingAverage()

Reply:
Hmm but you also dont say ema for you it's ewma ;p I found this confusing at first. Not sure I was just in need of descriptive naming

Reply:
how about creating an alias for this function? like:

var BOLL = BollingerBand()

because v2.BOLL() is also used in some external repositories.

Reply:
Renamed it back